packages/turf-projection/index.ts

```diff
@@ -82,25 +82,32 @@ function convert(geojson: any, projection: string, options: {mutate?: boolean} =
  *
  * @private
  * @param {Array<number>} lonLat WGS84 point
+ * @param {Object} [options] Optional parameters
+ * @param {boolean} [options.wrapLongitude=true] wrap logitude passing the meridian to a coordinated between -180 and 180
  * @returns {Array<number>} Mercator [x, y] point
  */
-function convertToMercator(lonLat) {
+function convertToMercator(lonLat, options: {wrapLongitude?: boolean} = {wrapLongitude: true}) {
+    options = options || {wrapLongitude: true};
```

Comment:
I'm not really familiar with Typescript, but I think this line is unnecessary as `options` has already a default value in the signature

Reply:
not familiar with typescript, 
I saw the same code for the options of `toMercator` / `toWgs84`,
here: 
https://github.com/Turfjs/turf/blob/master/packages/turf-projection/index.ts#L57
 
I guessed it was to handle the case where `options` is explicitly set to `null`.